src/pytestqt/qtbot.py

```diff
@@ -636,25 +636,85 @@ def keyToAscii(key):
             raise NotImplementedError("This method isn't available on PyQt5.")
         qt_api.QtTest.QTest.keyToAscii(key)
 
-    @staticmethod
-    def mouseClick(*args, **kwargs):
-        qt_api.QtTest.QTest.mouseClick(*args, **kwargs)
-
-    @staticmethod
-    def mouseDClick(*args, **kwargs):
-        qt_api.QtTest.QTest.mouseDClick(*args, **kwargs)
-
-    @staticmethod
-    def mouseMove(*args, **kwargs):
-        qt_api.QtTest.QTest.mouseMove(*args, **kwargs)
-
-    @staticmethod
-    def mousePress(*args, **kwargs):
-        qt_api.QtTest.QTest.mousePress(*args, **kwargs)
-
-    @staticmethod
-    def mouseRelease(*args, **kwargs):
-        qt_api.QtTest.QTest.mouseRelease(*args, **kwargs)
+    def mouseClick(self, widget, button, pos=None, modifiers=None):
+        if pos is None:
+            pos = widget.rect().center()
+        self.mouseMove(widget, pos)
```

Comment:
This doesn't seem right: if I issue a `mouseClick`, I don't expect a `mouseMove` to be issued.

Reply:
That's fair: `mousePress` and `mouseRelease` each take `pos` as an argument, so they should click the right spot without moving the cursor.